src/main/java/org/quiltmc/loader/impl/plugin/SolverErrorHelper.java

```diff
@@ -517,4 +482,145 @@ private static boolean reportDuplicateMandatoryMods(QuiltPluginManagerImpl manag
 
 		return true;
 	}
-}
\ No newline at end of file
+
+	/**
+	 * A solver error which might have multiple real errors merged into one for display.
+	 */
+	static abstract class SolverError {
+
+		/** Attempts to merge this error into the given error. This object itself shouldn't be modified.
+		 * 
+		 * @return True if the destination object was modified (and this was merged into it), false otherwise. */
+		abstract boolean mergeInto(SolverError into);
+
+		abstract void report(QuiltPluginManagerImpl manager);
+	}
+
+	static class UnhandledError extends SolverError {
+
+		final Collection<Rule> rules;
+
+		public UnhandledError(Collection<Rule> rules) {
+			this.rules = rules;
+		}
+
+		@Override
+		boolean mergeInto(SolverError into) {
+			return false;
+		}
+
+		@Override
+		void report(QuiltPluginManagerImpl manager) {
+			QuiltDisplayedError error = manager.theQuiltPluginContext.reportError(
+				QuiltLoaderText.translate("error.unhandled_solver")
+			);
+			error.appendDescription(QuiltLoaderText.of("error.unhandled_solver.desc"));
+			error.addOpenLinkButton(QuiltLoaderText.of("button.quilt_loader_report"), "https://github.com/QuiltMC/quilt-loader/issues");
+			error.appendReportText("Unhandled solver error involving the following rules:");
```

Comment:
Since there are still some super common cases that are unhandled (like `breaks`), this should probably go to the forum, not our issue tracker, since it's more of a support question than a "please fix this" question 99 times out of 100.

Reply:
Yes, good point.

Reply:
Okay, fixed.